test/mock-results.js

```diff
@@ -98,12 +111,6 @@ module.exports = function(stats, options) {
     ]
   };
 
-  if (options && options.skipPassedTests) {
```

Comment:
This got too complicated so had the few tests that used this just instantiate their own suites.